File: 백준/Bronze/11098. 첼시를 도와줘！/첼시를 도와줘！.py
T = int(input())
for i in range(T):
    price = 0
    name = ""
    n = int(input())
    for j in range(n):
        a, b = input().split()
        if int(a) > price :
            price = int(a)
            name = b
    print(name)

File: 백준/Bronze/11098. 첼시를 도와줘！/첼시를 도와줘！.cc
#include <iostream>

using namespace std;

int n, p, price = 0, resprice;
string name = "", resname;

int main(){
    cin >> n;
    for(int i = 0; i < n; i++){
        cin >> p;
        resprice = 0;
        resname = "";
        for(int j = 0; j < p; j++){
            cin >> price >> name;
            if(price > resprice){
                resname = name;
                resprice = price;
            }
        }
        cout << resname << endl;
    }
}